tpl_lessallrounder/index.php

```diff
@@ -26,9 +26,10 @@
 <html xmlns="http://www.w3.org/1999/xhtml" xml:lang="<?php echo $this->language; ?>" lang="<?php echo $this->language; ?>" dir="<?php echo $this->direction; ?>">
 <head>
 	<meta http-equiv="content-type" content="text/html; charset=utf-8" />
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
 	<jdoc:include type="head" />
 	<?php $id = isset($tpl->id) ? $tpl->id : $params->get('styleId'); ?>
-	<link href="<?php echo $path; ?>css/template<?php echo $params->get('useLESS') ? $id : ''; ?>.css" rel="stylesheet" type="text/css" media="all" />
+	<link href="<?php echo $path; ?>css/template<?php echo $params->get('useLESS',1) ? $id : ''; ?>.css" rel="stylesheet" type="text/css" media="all" />
```

Comment:
This actually has to stay without the default value. It's kind of a hack to detect if the params are already stored or not, using the default CSS if they are not yet stored.